### ISSUE TITLE
Using 3 monitors, the left side menu only appears in fullscreen when hovering over the left most monitor

### ISSUE THREAD

**Issue by EpicureanGit** [zaps166/QMPlay2#348]
![nvidiaxserversettings](https://user-images.githubusercontent.com/40360919/88234685-5b9d4800-cc47-11ea-830d-89ecd3235534.png)

The image is of my multi-monitor setup.

I am using QMPlay2-20.07.04. I tested with the appimage you provided and a build that was built for Slackware64 Current. The issue is found in both. When QMPlay2 is fullscreen on the HP Z24i I can access the menu by hovering over the left corner of the screen but it doesn't work for the other two monitors. On them nothing happens by hovering the mouse cursor over the left side of the screen. Other than that it works perfectly for me. It is my favorite video player! Thank you!

**Comment by zaps166** [comment]
I changed the behavior because of #311...

**Comment by EpicureanGit** [comment]
That was a feature that I liked. When I picked QMPlay2 to replace VLC I wanted a playlist that I could easily get to. The border wasn't a big issue and it would quickly change back unless it was qt5 5.9.6 (I think that is the one that introduced the bug that affected the playlist widget so it remained stuck). I reported about qt5 5.9.6. I have no idea if they ever fixed that branch of qt5 since I moved on. Please change the behavior back. When you click play or pause it shows up but it disappears as fast as the bottom with the play and pause. I don't understand how this could be a thing. I want the play and pause stuff removed too then. It shows up when I hover the mouse at the bottom of the screen. It has got to go. I'm kidding but that person is ridiculous. Otherwise if I play a video on another monitor I have to press F to toggle out of fullscreen and then I pick the video and then I toggle fullscreen again by pressing F. I just want to hover over the left side of the screen and pick something to watch on whatever monitor the video is playing on. The playlist and the play button quickly disappear. I don't understand how this could be a thing. In my setup above I used the Dell a lot, followed by the Acer. I rarely used the other one to play videos until now. Please change it back :D

**Comment by EpicureanGit** [comment]
I'm finding it impossible not to think of that person in #311  as a complete idiot regarding their asinine complaint. Can the playlist be toggled by hovering over the top part of the fullscreen instead as a compromise? I can't imagine the person hovering over the top of the screen while going for the play button at the bottom of the screen. I just don't get them.

**Comment by zaps166** [comment]
Could you try it now? The left sidebar can be displayed if bottom toolbar is visible.

**Comment by EpicureanGit** [comment]
It works if I hover over the play, stop, or previous buttons (for 1920x1200) then move my mouse cursor straight up above the bottom toolbar a little bit. For Ultra HD it works from the play button to the right side of the Fullscreen button and requires the same upward movement with the mouse cursor. I think getting to the playlist was more intuitive and easier to do before #311 but I'll gladly take this functionality. If it were me I'd revert the Playlist to before #311 and set it so it can be enabled or disabled under Widgets, or maybe allow it to be undocked into its own window. That is just my opinion though. Thank you for fixing it!

**Comment by zaps166** [comment]
> For Ultra HD it works from the play button to the right side of the Fullscreen button

Try to set DPR of 2.0 - `QT_SCALE_FACTOR=2` for 3840x2160

**Comment by zaps166** [comment]
You can also set DPR per screen, maybe try `QT_SCREEN_SCALE_FACTORS=1;2;1`

**Comment by EpicureanGit** [comment]
I got used to triggering the playlist on the left corner of the control bar around the first 3 buttons and moving the mouse up on the Dell monitor. After this update I had trouble toggling the playlist with the Dell monitor at all: https://github.com/zaps166/QMPlay2/commit/5337b4e5de0b32e93d173ff6bf3e8556f213a68d 

I changed the code to  "? qMax<int>(5, ceil(0.05 * (videoDock->isTouch ? 8 : 1) * width()))" to get it back to similar to how it was before the change.
